locales/en.default.schema.json

```
@@ -209,6 +209,20 @@
           }
         }
       }
+    },
+    "cart": {
+      "name": "Cart",
+      "settings": {
+        "add_to_cart_behavior": {
+          "label": "Add to cart behavior",
```

```suggestion
          "label": "Add to cart experience",
```

Adding this suggestion but I think I prefer the  neutral form of  `behavior`.  You can ignore!